### PR TITLE
feat(l10n): improve zh_CN translations

### PR DESCRIPTION
**Pull request by glepnir**
add some translations in `zh_CN.po`

### CODE REVIEW

File: src/nvim/po/zh_CN.UTF-8.po [hunk 1]
```diff
@@ -401,11 +401,11 @@ msgstr "已到段落结尾"
 
 #: ../edit.c:101
 msgid "E839: Completion function changed window"
-msgstr ""
+msgstr "E839: 补全函数更改窗口"
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:404]:
```suggestion
msgstr "E839: 补全函数更改了窗口"
```
Actually this error message is no longer used.

File: src/nvim/po/zh_CN.UTF-8.po [hunk 2]
```diff
@@ -401,11 +401,11 @@ msgstr "已到段落结尾"
...
 
 #: ../edit.c:102
 msgid "E840: Completion function deleted text"
-msgstr ""
+msgstr "E840: 补全函数删除文本"
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:408]:
```suggestion
msgstr "E840: 补全函数删除了文本"
```

File: src/nvim/po/zh_CN.UTF-8.po [hunk 3]
```diff
@@ -2671,7 +2674,7 @@ msgstr "E49: 无效的滚动大小"
 
 #: ../globals.h:1021
 msgid "E901: Job table is full"
-msgstr ""
+msgstr "E901: 工作表已经满"
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:2677]:
```suggestion
msgstr "E901: 任务表已经满"
```

File: src/nvim/po/zh_CN.UTF-8.po [hunk 4]
```diff
@@ -4978,49 +4990,51 @@ msgid ""
...
 #: ../regexp_nfa.c:239
 msgid "E865: (NFA) Regexp end encountered prematurely"
-msgstr ""
+msgstr "E865: (NFA) 过早的结束了正则表达式"
 
 #: ../regexp_nfa.c:240
 #, c-format
 msgid "E866: (NFA regexp) Misplaced %c"
-msgstr ""
+msgstr "E866: (NFA regexp) 位置错误 %c"
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:5003]:
```suggestion
msgstr "E866: (NFA regexp) 错误位置的 %c"
```

File: src/nvim/po/zh_CN.UTF-8.po [hunk 5]
```diff
@@ -3738,7 +3750,7 @@ msgstr ""
 
 #: ../memline.c:945
 msgid " has been damaged (page size is smaller than minimum value).\n"
-msgstr ""
+msgstr "已损坏（页面大小接近最小值）。\n"
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:3753]:
```suggestion
msgstr "已损坏（页面大小小于最小值）。\n"
```

File: src/nvim/po/zh_CN.UTF-8.po [hunk 6]
```diff
@@ -44,7 +44,7 @@ msgstr "[Quickfix 列表]"
 
 #: ../buffer.c:94
 msgid "E855: Autocommands caused command to abort"
-msgstr ""
+msgstr "E855: Autocommands 导致命令被停止"
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:47]:
```suggestion
msgstr "E855: 自动命令导致命令被停止"
```

**Review by zeertzjq — COMMENTED** [review]
You can take a look at Vim's translation file <https://github.com/vim/vim/blob/master/src/po/zh_CN.UTF-8.po>, which was updated last month.

**Comment by glepnir** [comment]
oh I forget vim's translate

**Comment by glepnir** [comment]
done. but also have some translations that differ from vim. I think it's more correct

File: src/nvim/po/zh_CN.UTF-8.po [hunk 7]
```diff
@@ -32,7 +32,7 @@ msgstr "选项参数后的内容无效"
 
 #: ../api/private/helpers.c:204
 msgid "internal error: unknown option type"
-msgstr ""
+msgstr "内部错误： 未知的选项类型"
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:35]:
```suggestion
msgstr "内部错误：未知的选项类型"
```

File: src/nvim/po/zh_CN.UTF-8.po [hunk 8]
```diff
@@ -4978,49 +4992,51 @@ msgid ""
 "E864: \\%#= can only be followed by 0, 1, or 2. The automatic engine will be "
 "used "
 msgstr ""
+"E864: \\%#= 后面只能是0，1，或者2。自动引擎将会 "
+"使用 "
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:4996]:
```suggestion
"E864: \\%#= 后面只能是0，1，或者2。自动引擎将会被使用"
```

**Comment by glepnir** [comment]
thanks for fixup

File: src/nvim/po/zh_CN.UTF-8.po [hunk 9]
```diff
@@ -5662,7 +5683,7 @@ msgstr "E47: 读取错误文件失败"
 #. file.
 #: ../spell.c:11575
 msgid "E783: duplicate char in MAP entry"
-msgstr ""
+msgstr "E783: 重复的字符在 MAP 中"
```
Comment (zeertzjq) [inline src/nvim/po/zh_CN.UTF-8.po:5686]:
```suggestion
msgstr "E783: MAP 条目中有重复的字符"
```

**Pull request merged**